pages.ru/android/dalvikvm.md

```diff
@@ -0,0 +1,8 @@
+# dalvikvm
+
+> Виртуальная машина Android Java.
+> Больше информации: <https://source.android.com/devices/tech/dalvik>.
+
+- Запустить Java-программу:
+
+`dalvikvm -classpath {{path/to/file.jar}} {{classname}}`
```

Comment:
```suggestion
`dalvikvm -classpath {{путь/к/файлу.jar}} {{classname}}`
```

- This change is optional - no corresponding style guide requirement exists.
- It's better to provide placeholder translations when possible.